source/fundamentals/data-formats/pojo-customization.txt

```diff
@@ -300,14 +300,18 @@ package:
        nested within the field.
 
    * - ``BsonExtraElements``
-     - Specifies the POJO field on which to deserialize any elements that are
+     - Specifies the POJO field on which to deserialize all elements that are
        not mapped to a field. The POJO field must be one of the following
        types:
 
        - `Document <{+api+}/apidocs/bson/org/bson/Document.html>`__
        - `BsonDocument <{+api+}/apidocs/bson/org/bson/BsonDocument.html>`__
        - Map<String, Object>
 
+       When serializing a POJO through an insert or replace operation, the
```

Comment:
I don't think this is true.  Rather, I think each field in the Map is stored at the same level as the other fields in the POJO.

Reply:
Sorry, I was focused on updating the other part and forgot to update this mention.